src/tests/illumidesk/authenticators/test_lti11_authenticator.py

```diff
@@ -173,6 +175,9 @@ def _change_flag():
 
 @pytest.mark.asyncio
 async def test_authenticator_invokes_validator_with_decoded_dict():
+    """
+    Does the authentictor call the validator?
+    """
     with patch.object(LTI11LaunchValidator, 'validate_launch_request', return_value=True) as mock_validator:
```

Comment:
it seems that the name of the test not corresponds with the assert. The test says that call of validator class is expected but there are other checks

Reply:
I'm not sure about this one. It may come up in the diff because things got moved around with the workspace tests but was left as-was.